xbmc/pvr/epg/EpgInfoTag.h

```diff
@@ -9,6 +9,7 @@
 #pragma once
 
 #include "XBDateTime.h"
+#include "addons/kodi-addon-dev-kit/include/kodi/c-api/addon-instance/pvr/pvr_epg.h"
```

Comment:
Are you sure the new include is needed in the header file? I cannot see anything there that requires the include.

Reply:
Was a bit wrongly added there as `EpgInfoTag.h` is mostly the counterparts of `pvr_epg.h`, is removed, was only then a add needed on `xbmc/interfaces/json-rpc/FileItemHandler.cpp`